R/calcPOP.R

```diff
@@ -1,9 +1,22 @@
-calcPOP <- function(scenario = 'SSP2') {
+#' calcPOP
+#'
+#' The SSP data filtered by pop
```

Comment:
better phrase the description like this: "Derive population data based on the SSP scenarios"

Reply:
ok